circuits/cpp/barretenberg/cpp/src/barretenberg/benchmark/relations_bench/barycentric.bench.cpp

```diff
@@ -0,0 +1,29 @@
+#include "barretenberg/ecc/curves/bn254/fr.hpp"
+#include "barretenberg/polynomials/barycentric.hpp"
+#include <benchmark/benchmark.h>
+
+using namespace benchmark;
+
+namespace {
+auto& engine = numeric::random::get_debug_engine();
+}
+
+using FF = barretenberg::fr;
+using barretenberg::Univariate;
+using barretenberg::BarycentricData;
+
+
+namespace proof_system::relation::benchmark {
+
+void extend_2_to_6(State& state) noexcept
```

Comment:
Used this to compare the optimization I put in `BarycentricData` to make extension from degree 2 very efficient. It gives approximately a 10x improvement, in one measurement, 417ns to 40ns.